company-profiles/ombu-labs.md

```diff
@@ -0,0 +1,35 @@
+# Ombu Labs
+
+## Company blurb
+
+We are not yet another software factory, we are the lean software boutique.
+We design and develop products for the web. We write as little code as possible
+to validate your hypothesis.
+
+We are an agile team of designers and developers who write lean software to grow
+your business or launch your new product.
+
+## Company size
+
+5-10
+
+## Remote status
+
+Our Headquarters is in Mt. Airy, Philadelphia, but team members
+spread from Argentina to Brazil.
```

Comment:
Argentina and Brazil are neighbors!  Should this be more like "US to Argentina" ?

Reply:
@nylen Good point 👍